### DISCUSSION TITLE
Show / Hide elements if Midi-Player is active

### DISCUSSION THREAD

**Discussion by PJOttCanHome** [jazz-soft/midi-player#5] [Ideas]
I have a webpage with MIDI file playback setup for it.

If a browser has this extension installed, and it finds the A,OBJECT,EMBED,etc element to manipulate can you also have it show/hide elements based on that?

i.e. using classname
```
<div class="jsmidiplayerhide"></div>
<div class="jsmidiplayershow"></div>

```
That way I can have certain things show up on my page if your extension does its thing.

Or, is there another way to do this already?

**Comment by jazz-soft** [comment]
I'm not sure I understand the idea. Could you please explain it in more details?
You can enable MIDI Player on your page even if the visitor does not have the extension installed.

**Reply by PJOttCanHome** [reply]
How do you enable it without the extension?  Maybe some notes on that could help

The idea is that, if you find an element to replace with MIDI Player on the page, then you can automatically hide/show elements with a certain classname after that so my page can show a "Hey, Kids, this page is using Jazz-Soft's MIDI Player Extension" and hide the "I recommend you install Jazz-Soft's MIDI Player Extension" notice.  Something like that.

I think in jQuery: $('.jsmidiplayershow').show()

**Reply by jazz-soft** [reply]
Please check https://github.com/jazz-soft/JZZ-gui-Player

**Reply by PJOttCanHome** [reply]
Thank you very much.  That will do nicely!
